src/core/contrib.js

```diff
@@ -24,9 +24,12 @@ function findUserURLs(...thingsWithUsers) {
 }
 
 async function toHTML(urls, editors, element, headers) {
-  const args = await Promise.all(urls.map(url => fetch(url, { headers })));
-  const names = args
-    .map(([user]) => user.name || user.login)
+  const args = await Promise.all(
+    urls.map(url => fetch(new Request(url, { headers })))
+  );
+  const argsResolved = await Promise.all(args.map(arg => arg.json()));
```

Comment:
Can we do this `.json()` in the `args` above?

Reply:
Good point.

Reply:
Right. Done. :-)